docs/admin-guide/zope-manager-users.md

```diff
@@ -107,3 +107,45 @@ When you run the script, if the user already exists:
     Maybe the user already exists and nothing is done then.
     Or the implementation does not give info when it succeeds.
     ```
+### `addzopeuser` script in dockerized setup
+
+#### Prerequisites
+-   A running docker swarm stack from the full featured docker based Plone Volto deployment 
+-   Created by current Cookieplone project template including backend, frontend, postgres, traefik, varnish:
+
+#### Create a new Zope Manager User with a non existing user-ID
+
+-   Enter the host via ssh as root
+-   list the running containers: docker ps
+-   enter a shell in the first backend container listed: `docker exec -it [[4-digitPartOfID]] bash`
+-   run command in the app folder: `./docker-entrypoint.sh bin/addzopeuser -c /app/etc/relstorage.conf userid password`
+
+Response: `User userid created`
+The user is now available in the ZMI root at at /acl_users/users/manage_users
+
+#### Hown to access the ZMI root from via basicauth and the new user 
+In the Volto page root use the url `/ClassicUI/aq_parent/acl_users/users/manage_users`
```

Comment:
This is a path, not a URL. It's relative to the hostname where the Docker container is served via Traefik.